cmds/modules/noded/main.go

```diff
@@ -40,6 +41,11 @@ const (
 	eventsBlock     = "/tmp/events.chain"
 )
 
+type Diagnostics struct {
```

Comment:
Not the right place, should be in client/node.go (next to zos.statistics.get). cmds is only for commands (binaries)